alf/trainers/policy_trainer.py

```diff
@@ -34,9 +34,38 @@
 from alf.utils.summary_utils import record_time
 
 
+class TrainerProgress(nn.Module):
+    def __init__(self):
```

Comment:
It might be better to use ```global_counter``` for this purpose instead of introducing another counter.

Reply:
global_counter is not equal to iter_num because when update_every_minibatch is true it's a multiple of iter_num. Also currently iter_num is not checkpointed, which is inconsistent with env_steps.